lib/args.js

```diff
@@ -37,6 +37,12 @@ function buildYargs(args = null) {
       describe: 'File to write the metadata in',
       type: 'string'
     })
+    .option('comments', {
```

Comment:
I'd rather let this be `--check-comments` with no aliases, in case we need the `c` later. This does not seem to be very in demand.

Reply:
Your desires are my commands!